package/gluon-mesh-babel/check_site.lua

```diff
@@ -1,6 +1,8 @@
 need_string_match(in_domain({'node_prefix6'}), '^[%x:]+/64$')
 need_string_match(in_domain({'node_client_prefix6'}), '^[%x:]+/64$')
 
+need_string_match(in_domain({'clat_range'}), '^[%x:]+/48$', false)
```

Comment:
Why does babel require this? And why is this check more specific that the one in `gluon-464xlat-clat`? Shouldn't `gluon-mesh-babel` just depend on `gluon-464xlat-clat` instead?

Reply:
Babel needs to know which routes (from which IP ranges) should be redistributed.

Reply:
But the clat functionality is only there, when the clat package is actually included, no?

Reply:
If a node does not use the CLAT (whyever) it should still redistribute the route. I agree that the site-check fits better in the clat-package itself. There is no clear solution for this, but I guess nobody will be mixing firmwares with and without clat-support. Based on that it is fine to move it to the clat-package. I don't think gluon-mesh-babel should depend on the clat package.